conflict-transformation.md

```diff
@@ -1,22 +1,21 @@
-
 # Conflict Transformation Process
 
->The idea is not to eliminate conflict…The aim is to transform it. 
-                                                    *Diane Musho Hamilton*
-
+>The idea is not to eliminate conflict... The aim is to transform it. 
+>
+>*--Diane Musho Hamilton*
 
-All member-workers, employees and contractors of Hypha Worker Co-operative must agree to follow [Hypha’s Code of Conduct](https://handbook.hypha.coop/coc.html), [principles of being a good member](https://handbook.hypha.coop/member-workers.html) and uphold the [co-op’s values](https://handbook.hypha.coop/vision.html) when working together. These form the guidelines for our work together.
```

Comment:
@writerly03 @LexaMichaelides see how we link to internal pages

Reply:
For the other minor edits, I don't know what character you're using instead of an apostrophe. Pls show me tomorrow?